src/ppom-controller.ts

```diff
@@ -453,6 +466,18 @@ export class PPOMController extends BaseControllerV2<
     );
   }
 
+  /**
+   * The function check to ensure that file path can contain only alphanumeric characters and a dot character (.) or slash (/).
+   *
+   * @param filePath - Path of the file.
+   */
+  #checkFilePath(filePath: string) {
+    const filePathRegex = /^[\w./]+$/u;
+    if (!filePathRegex.test(filePath)) {
```

Comment:
I am always a bit concern with using `regex.test`.. If the regex has a global or sticky flag, then it will be statefull (meaning that running the `test` twice will produce different results, as each time you run test it will update the regex lastIndex).

I know that here we are not using any of those flags, but I tend to prefer using `string.match(regex)`.

What do you think?

Reply:
PR is updated